src/seals/testing/envs.py

```diff
@@ -155,13 +157,13 @@ def test_seed(
     env.action_space.seed(0)
     actions = [env.action_space.sample() for _ in range(rollout_len)]
     # With the same seed, should always get the same result
-    seeds = env.seed(42)
+    seeds = env.reset(seed=42)
```

Comment:
`env.reset` does not return a list of seeds.

Reply:
Quoting @RedTachyon from [here](https://github.com/HumanCompatibleAI/seals/pull/72#discussion_r1250594110):

> This is probably from just replacing `s/env.seed/env.reset/g`, but `env.reset` doesn't reset the seeds, it should return `obs, info`, which happens to also be a tuple.
> Probably narrow down the assertions below, that it should now always be a 2-tuple, with the second item being a dict.